x/programs/rust/sdk-macros/src/lib.rs

```diff
@@ -357,7 +357,7 @@ pub fn state_schema(input: TokenStream) -> TokenStream {
 
                     const _: fn() = || {
                         #[doc(hidden)]
-                        struct TypeWithoutPadding([u8; 1 + ::core::mem::size_of::<#key_type_name>()]);
+                        struct TypeWithoutPadding([u8; 3 + ::core::mem::size_of::<#key_type_name>()]);
```

Comment:
can we not hardcode this here and potentially use the `STATE_MAX_CHUNKS` const defined below?

Reply:
I wonder if we can actually re-use some code here. Let me try something...

Reply:
Fixed